include/triton/Target/LLVMIR/LLVMIRTranslation.h

```diff
@@ -2,6 +2,7 @@
 #define TRITON_TARGET_LLVMIRTRANSLATION_H
 #include "llvm/ADT/StringRef.h"
 #include <memory>
+#include <string>
```

Comment:
is this change on purpose？

Reply:
Yes, I think it is better to explicit list all the depending header files.